empathy.iml

```diff
@@ -2,7 +2,9 @@
 <module type="JAVA_MODULE" version="4">
   <component name="NewModuleRootManager" inherit-compiler-output="true">
     <exclude-output />
-    <content url="file://$MODULE_DIR$" />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/.bach" isTestSource="false" />
```

Comment:
Simple (legacy class-path) source folder to make `BuildSenseNine` runnable from within the IDE.
After running `jshell build.jsh` to bootstrap (read: download) Bach...